website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step08-apply-helm-chart.sh

```diff
@@ -4,8 +4,4 @@ helm upgrade --install karpenter oci://public.ecr.aws/karpenter/karpenter --vers
   --set settings.aws.clusterName=${CLUSTER_NAME} \
   --set settings.aws.defaultInstanceProfile=KarpenterNodeInstanceProfile-${CLUSTER_NAME} \
   --set settings.aws.interruptionQueueName=${CLUSTER_NAME} \
-  --set controller.resources.requests.cpu=1 \
```

Comment:
These should be kept. This is effectively the "default" resource requests and limits that we expect a user to be aware of in the getting started guide. Otherwise, the controller requests/limits are completely wide-open

Reply:
I left a comment on the PR dropping this in the chart, where I think having a reasonable defaults might be a better approach. Happy to leave it here, but feels more prone to human error :shrug:

Reply:
I disagree. This surfaces it so that the user is more aware that they are setting explicit values. IMO, this is better than not including it and not making the user aware that they aren't set.

Reply:
Not sure I was clear enough. So since we don't have defaults set in the Chart anymore I agree we should have this in the script. My comment was making it default and giving users an option to override it here.

Reply:
Also restored those params in this PR